rest/route/pod.go

```diff
@@ -96,3 +96,49 @@ func (h *podPostHandler) Run(ctx context.Context) gimlet.Responder {
 
 	return responder
 }
+
+////////////////////////////////////////////////
+//
+// GET /rest/v2/pods/{pod_id}
+
+type podGetHandler struct {
+	env   evergreen.Environment
+	sc    data.Connector
+	podID string
+}
+
+func makeGetPod(env evergreen.Environment, sc data.Connector) gimlet.RouteHandler {
+	return &podGetHandler{
+		env: env,
+		sc:  sc,
+	}
+}
+
+func (h *podGetHandler) Factory() gimlet.RouteHandler {
+	return &podGetHandler{
+		env: h.env,
+		sc:  h.sc,
+	}
+}
+
+// Parse fetches the pod ID from the HTTP request.
+func (h *podGetHandler) Parse(ctx context.Context, r *http.Request) error {
+	h.podID = gimlet.GetVars(r)["pod_id"]
+	return nil
+}
+
+// Run creates a new resource based on the Request-URI and JSON payload.
```

Comment:
I think this comment also needs updating.

Reply:
Also fixed!